locale/model_attributes.rb

```diff
@@ -3,7 +3,7 @@
 # TRANSLATORS: "Table name" or "Table name|Column name" for error messages
 _('Architecture|Name')
 # TRANSLATORS: "Table name" or "Table name|Column name" for error messages
-_('Audit')
+_('»Audit«')
```

Comment:
This one looks odd, where did the `»` come from?

Reply:
It also happens if I run `rake locale:find` without this PR change.
It comes from here:
https://github.com/theforeman/foreman/blob/develop/app/models/concerns/audit_extensions.rb#L26

Reply:
But why we have the `>> <<` added to the mix now?

Reply:
Looks like something calls `_(Audit.humanize_class_name)` so you end up with `_(_())`

Reply:
I believe it could be because of: #8999

Reply:
The `store_model_attributes` task calls:
`f.puts("_('#{model.humanize_class_name}')")`
https://github.com/grosser/gettext_i18n_rails/blob/184ad2afa628d8f11ee674cd6e1c50f86e020410/lib/gettext_i18n_rails/model_attributes_finder.rb#L12
Looks like Ondrej's pr solves it

Reply:
Rebased with Ondrejs pr